### PR TITLE
Minor clarity edits.

### PR DESCRIPTION
**Pull request by NightJar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,9 +98,9 @@ protected function init()
 
 We use `themedCSS` and `themedJavascript` to auto-locate the resource based on the current theme. That way, if the theme ever changes, we don't have to update the path.
 
-The only resources we haven't included are the html5 shim that is conditionally included for IE8, and the Google font that is loaded from an absolute path.
+The only resource we haven't included is the html5 shim that is conditionally included for IE8, and the Google font that is loaded from an absolute path.
```
Comment (robbieaverill) [inline README.md]: I reckon it fits better the way it was - thoughts?

Comment (NightJar) [reply README.md]: It's a single resource (one file), referenced in the singular (shim, not shims), that's why I altered it.

Comment (robbieaverill) [reply README.md]: Does the google font not count as a resource? It does in my head

Comment (NightJar) [reply README.md]: lol, brain, how do you even work? Totally just skipped over that xD
Reminds me of those posters that go:

```
This was silly and
and rather illogical
```

But then again...
https://github.com/silverstripe/silverstripe-lessons-v4/pull/6/files/2b8d9bb37d23b63af615557e0548275e0f848ce4#diff-04c6e90faac2675aa89e2176d2eec7d8R84

Seems the bit I need to update is actually to remove the reference to the Google font.

**Review by unclecheese — APPROVED** [review]

**Pull request merged**